EPPlus/ExcelCellBase.cs

```diff
@@ -212,8 +228,20 @@ private static string ToR1C1(string part, int row, int col, int rowIncr, int col
         /// <param name="rowIncr"></param>
         /// <param name="colIncr"></param>
         /// <returns></returns>
-        private static string ToAbs(string part, int row, int col, int rowIncr, int colIncr)
+        private static string ToAbs(string part, int row, int col, int rowIncr, int colIncr, bool isNoRange = false)
         {
+            // Allow for static R1C1 style column references or static column range references
+            string pattern = @"^C([0-9]+)$";
+            if (Regex.IsMatch(part, pattern))

```

Comment:
Regex is very slow. At least consider Compiled Regex (https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices#interpreted-vs-compiled-regular-expressions). 
But I think the pattern is too simple for regex, you can just use char.IsDigit(character) in simple for loop which is the fastest solution